exercism/hamming/hamming.js

```diff
@@ -0,0 +1,18 @@
+var Hamming = function() {}
+
+Hamming.prototype.compute = function (string1, string2) {
```

Comment:
Sorry it might be my test again, but I tried running this and it tells me that compute is not a function.